### PR TITLE
Refactor calc rho

### PR DESCRIPTION
**Pull request by profxj**
Refactors calculation of impact parameter to 
top level of pyigm.

Tests and other code were updated too.
No new docs.

### CODE REVIEW

**Comment by profxj** [comment]
hold up.  want to swap ordering..

**Comment by profxj** [comment]
Reopening.

File: pyigm/cgm/utils.py [hunk 1]
```diff
@@ -15,73 +15,228 @@
...
     cosmo : astropy.cosmology
-    Galactic : bool, optional
-      Calculate for our Galaxy!
+    ang_sep : Quantity, optional
     correct_lowz : bool, optional
-      Apply corrections for the local universe, as desired
-      Follows Mould et al. 2000
-    ang_sep : Angle or Quantity
-      Input angular separation
-      May speed up calculation
```
Comment (jnburchett) [inline pyigm/cgm/utils.py:36]: Why remove all this?  Some potentially very useful information here...

**Review by jnburchett — APPROVED** [review]
Looks good except for couple docstring changes.

File: pyigm/cgm/utils.py (hunk 1 above)
Comment (profxj) [reply pyigm/cgm/utils.py:36]: Decided to use **kwargs instead

**Comment by profxj** [comment]
merging

**Pull request merged**